logic/src/commonMain/kotlin/com/wire/kalium/logic/data/conversation/Conversation.kt

```diff
@@ -61,7 +61,13 @@ sealed class ConversationDetails(open val conversation: Conversation) {
 
 class MembersInfo(val self: Member, val otherMembers: List<Member>)
 
-class Member(override val id: UserId) : User()
+class Member(override val id: UserId, val role: Role) : User() {
+    sealed class Role{
+        object Member: Role()
+        object Admin: Role()
+        data class Unknown(val name: String): Role()
+    }
+}
 
 sealed class MemberDetails {
```

Comment:
Shouldn't the role be also included in `MemberDetails`? 🤔

Reply:
I was considering changing the `MemberDetails` to return a list of admin and Members instead of adding role to `Self`, `Other`

Reply:
Anyway, I will do this change in the next PR to avoid getting this one too big